### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by rwwagner90**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (nummi) [inline .github/ISSUE_TEMPLATE/bug_report.md]: bug

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
+
```
Comment (nummi) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Seems like a lot of noise for Desktop and Smartphone. Would this work:

```
**Environment**
Ember version, browser, etc.
```

**Comment by nummi** [comment]
We could also remove `ember-inspector/ISSUE_TEMPLATE.md`.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (rwwagner90) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yeah, works for me

**Comment by nummi** [comment]
👍🏽

**Pull request merged**